readme.md

```diff
@@ -46,13 +46,29 @@ module.exports = {
   mongodbMemoryServerOptions: {
     binary: {
       version: '4.0.3',
-      skipMD5: true
+      skipMD5: true,
     },
     instance: {
-      dbName: 'jest'
+      dbName: 'jest',
+    },
+    autoStart: false,
+  },
+};
+```
+
+To use separate database for each jest worker pass the `separateMongoInstances: true` (doesn't create `process.env` variable when using this option):
```

Comment:
```suggestion
To use separate database for each jest worker pass the `useSharedDBForAllJestWorkers: false` (doesn't create `process.env` variable when using this option):
```

Maybe let's rename it to something like this. What do you think?

Reply:
If you mean that jest-mongodb will start distinct database by default, I think it can result in unexpected performance decrease for ones who updated package but still not familiar with this feature.

Reply:
It will have the same behaviour, just naming changed and default to value of "true" rather than false

Reply:
Yes. What I mean is just **renaming the variable**.